benchmark_runner/grafana/perf/jsonnet/g.libsonnet

```diff
@@ -0,0 +1,2 @@
+// Cannot import it in main.libsonnet; it must be in a separate file, g.libsonnet for more details: https://github.com/grafana/grafonnet/tree/main/examples/terraform
```

Comment:
That's just a pointer to an example.  Note that attempting to import it inside main.libsonnet results in an error.

Reply:
Updated